v0/destinations/vero/config.js

```diff
@@ -0,0 +1,50 @@
+const { getMappingConfig } = require("../../util");
+
+const BASE_URL = "https://api.getvero.com/api/v2";
+
+const CONFIG_CATEGORIES = {
+  IDENTIFY: {
+    name: "VeroIdentifyConfig",
+    type: "identify",
+    endpoint: `${BASE_URL}/users/track`,
+    method: "POST"
+  },
+  TRACK: {
+    name: "VeroTrackConfig",
+    type: "track",
+    endpoint: `${BASE_URL}/events/track`,
+    method: "POST"
+  },
+  ALIAS: {
+    name: "VeroAliasConfig",
+    type: "alias",
+    endpoint: `${BASE_URL}/users/reidentify`,
+    method: "PUT"
+  },
+  TAGS: {
+    name: "VeroTagsConfig",
```

Comment:
VeroTagsConfig is not only used for tag payload but for others we should name it to something meaningful

Reply:
Changed to VeroSimpleConfig